js/plates-model/field.ts

```diff
@@ -368,9 +370,14 @@ export default class Field extends FieldBase<Field> {
         this.crust.addSediment(0.02 * timestep);
       }
     }
-    if (this.volcanicAct?.active && this.volcanicAct.erupting) {
-      this.crust.addVolcanicRocks(this.volcanicAct.intensity * timestep * VOLCANIC_ACTIVITY_STRENGTH);
-      this.volcanicAct.deformingCapacity -= timestep;
+    if (this.volcanicAct?.active) {
+      if (this.volcanicAct.erupting) {
+        // This will tall and more spread out volcanic peaks.
```

Comment:
```suggestion
        // This will heighten and spread out volcanic peaks.
```

Reply:
I think I meant to type "create" there, but this works too. :wink: